### PR TITLE
fix: webhooks are requests so readOnly doesn't make sense

### PR DESCRIPTION
**Pull request by adamaltman**

### CODE REVIEW

File: openapi/components/requestBodies/Webhooks/Customer.yaml [hunk 1]
```diff
@@ -5,15 +5,13 @@ content:
         customerId:
           description: The customer ID
           type: string
-          readOnly: true
         eventType:
           $ref: ../../schemas/GlobalWebhookEventType.yaml
         _embedded:
-          $ref: ../../schemas/Embeds/CustomerEmbed.yaml
+          $ref: ../../schemas/Customer.yaml
```
Comment (slavcodev) [inline openapi/components/requestBodies/Webhooks/Customer.yaml:11]: This is incorrect schema. the `_embedded` is a map not an object.
```yaml
_embedded:
  type: object
  properties:
    customer:
       $ref: ../../schemas/Customer.yaml
```

Comment (slavcodev) [reply openapi/components/requestBodies/Webhooks/Customer.yaml:11]: Not related to this PR tho

**Review by slavcodev — DISMISSED** [review]

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/components/requestBodies/Webhooks/Customer.yaml (hunk 1 above)
Comment (ashkarpetin) [reply openapi/components/requestBodies/Webhooks/Customer.yaml:11]: This seems wrong change, CustomerEmbed is a map

Comment (slavcodev) [reply openapi/components/requestBodies/Webhooks/Customer.yaml:11]: Indeed

Comment (adamaltman) [reply openapi/components/requestBodies/Webhooks/Customer.yaml]: Thank you.

**Review by ashkarpetin — APPROVED** [review]

**Review by slavcodev — APPROVED** [review]

**Pull request merged**